consul/config_templates/consul.yaml

```diff
@@ -6,112 +6,98 @@ apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
   name: consul-consul-server
-  namespace: consul
+  namespace: {{.namespace}}
   labels:
     app: consul
     chart: consul-helm
-    heritage: Tiller
+    heritage: Helm
     release: consul
 spec:
-  maxUnavailable: 1
+  maxUnavailable: 0
```

Comment:
Why change the value of `maxUnavailable` to `0`?

Reply:
From the documentation in the values file:
    # maxUnavailable will default to (n/2)-1 where n is the number of
    # replicas. If you'd like a custom value, you can specify an override here.
If the number of replicas is 1, then this results in 0.
This would allow a disruptive change to occur in a configuration with 1 replica like here, e.g. when upgrading consul.

Reply:
This is a nice catch. As you said, the number of `replicas` and `maxUnavailable ` should be as a group to set to avoid unknown issue